.storybook/.babelrc

```diff
@@ -1,12 +1,10 @@
 {
   "plugins": [
     ["@babel/plugin-proposal-class-properties", { "loose": true }],
-    "lodash"
+    ["@babel/plugin-proposal-private-methods", { "loose": true }],
+    ["@babel/plugin-proposal-private-property-in-object", { "loose": true }],
```

Comment:
I thought Storybook would configure these plugins itself, weird.

Reply:
Very happy that the warnings are gone though! 🎉

Reply:
Yeah also not sure why this was necessary, but I've tried a bunch of stuff and this is the only thing that actually silenced the warnings 😅 and the same config tweaks in the other babel configs were required to avoid the console warning right after the storybook was built—I guess it first uses the sb babel config to build sb itself, and then the package (or root) babel config to build components 🤷